src/utils/collection.ts

```diff
@@ -60,6 +60,25 @@ const formatConsentValues = (publisherConsents: Consent[] | null = [], customerC
   if (!publisherConsents || !customerConsents) {
     return {};
   }
+
+  const values: Record<string, string | boolean> = {};
+
+  publisherConsents?.forEach((publisherConsent) => {
+    const consent = customerConsents?.find((customerConsent) => customerConsent.name === publisherConsent.name);
+
+    if (consent) {
+      const value = publisherConsent.isCustomRegisterField === true ? consent.value ?? '' : consent.state === 'accepted';
```

Comment:
Because consent have two possible answers: `accepted` and `declined`. While custom register fields have multiple possibilities: `on`, `off` which are interchangeable with `true` and `false`, and practically any string possible.